hypothesis-python/src/hypothesis/vendor/pretty.py

```diff
@@ -801,7 +802,11 @@ def _repr_dataframe(obj, p, cycle):  # pragma: no cover
 
 
 def _repr_enum(obj, p, cycle):
-    p.text(type(obj).__name__ + "." + obj.name)
+    tname = type(obj).__name__
+    if isinstance(obj, Flag):
+        p.text(" | ".join(f"{tname}.{x.name}" for x in type(obj) if x & obj == x))
```

Comment:
@Zac-HD I just realized is is possible to construct empty flags e.g. `Options(0)`. These are unnamed, and not constructible be name. Maybe
```
p.text(" | ".join(...) or f"{tname}(obj.value)")
```
Too late for this PR, sorry.

Reply:
Oooh, wow, that would have been _bizzare_ if anyone hit it in the wild!  Will fix.